libs/language-server/src/lib/ast/wrappers/index.ts

```diff
@@ -3,13 +3,24 @@
 // SPDX-License-Identifier: AGPL-3.0-only
 
 // eslint-disable-next-line import/no-cycle
-export * from './util';
 export * from './value-type';
 
-export * from './ast-node-wrapper';
-export * from './cell-range-wrapper';
+/* Exports of directory './typed-object'; only exporting types to force usage of WrapperFactory */
```

Comment:
Fix comment

Reply:
Done in [1dac8ca](https://github.com/jvalue/jayvee/pull/536/commits/1dac8cafe69fb7e920efce243e7ca4c5220f0575)